drivers/regulator/regulator_axp192.c

```diff
@@ -0,0 +1,398 @@
+#define AXP192_REG_LDO23_VOLTAGE         (0x28u)
+
+/* DCDC1 parameters */
+#define AXP192_DCDC1_VOLTAGE_REG_MASK   (0x7Fu)
+#define AXP192_DCDC1_VOLTAGE_REG_BITPOS (0u)
+#define AXP192_DCDC1_VOLTAGE_RANGE_MAX  (AXP192_DCDC1_VOLTAGE_REG_MASK)
+#define AXP192_DCDC1_VOLTAGE_RANGE_MIN  (0x00u)
+#define AXP192_DCDC1_VOLTAGE_MIN_UV     (700000u)
+#define AXP192_DCDC1_VOLTAGE_MAX_UV     (3500000u)
+#define AXP192_DCDC1_VOLTAGE_STEP_UV    (25000u)
+#define AXP192_DCDC1_ENABLE_REG_MASK    (0x01u)
+#define AXP192_DCDC1_ENABLE_REG_VAL     (0x01u)
+
+/* DCDC2 parameters */
+#define AXP192_DCDC2_VOLTAGE_REG_MASK   (0x3Fu)
+#define AXP192_DCDC2_VOLTAGE_REG_BITPOS (0u)
+#define AXP192_DCDC2_VOLTAGE_RANGE_MAX  (AXP192_DCDC2_VOLTAGE_REG_MASK)
+#define AXP192_DCDC2_VOLTAGE_RANGE_MIN  (0x00u)
+#define AXP192_DCDC2_VOLTAGE_MIN_UV     (700000u)
+#define AXP192_DCDC2_VOLTAGE_MAX_UV     (2275000u)
+#define AXP192_DCDC2_VOLTAGE_STEP_UV    (25000u)
+#define AXP192_DCDC2_ENABLE_REG_MASK    (0x10u)
+#define AXP192_DCDC2_ENABLE_REG_VAL     (0x10u)
+
+/* DCDC3 parameters */
+#define AXP192_DCDC3_VOLTAGE_REG_MASK   (0x7Fu)
+#define AXP192_DCDC3_VOLTAGE_REG_BITPOS (0u)
+#define AXP192_DCDC3_VOLTAGE_RANGE_MAX  (AXP192_DCDC3_VOLTAGE_REG_MASK)
+#define AXP192_DCDC3_VOLTAGE_RANGE_MIN  (0x00u)
+#define AXP192_DCDC3_VOLTAGE_MIN_UV     (700000u)
+#define AXP192_DCDC3_VOLTAGE_MAX_UV     (3500000u)
+#define AXP192_DCDC3_VOLTAGE_STEP_UV    (25000u)
+#define AXP192_DCDC3_ENABLE_REG_MASK    (0x02u)
+#define AXP192_DCDC3_ENABLE_REG_VAL     (0x02u)
+
+/* LDO2 parameters */
+#define AXP192_LDO2_VOLTAGE_REG_MASK   (0xF0u)
+#define AXP192_LDO2_VOLTAGE_REG_BITPOS (4u)
+#define AXP192_LDO2_VOLTAGE_RANGE_MAX  (0x0Fu)
+#define AXP192_LDO2_VOLTAGE_RANGE_MIN  (0x00u)
+#define AXP192_LDO2_VOLTAGE_MIN_UV     (1800000u)
+#define AXP192_LDO2_VOLTAGE_MAX_UV     (3300000u)
+#define AXP192_LDO2_VOLTAGE_STEP_UV    (100000u)
+#define AXP192_LDO2_ENABLE_REG_MASK    (0x04u)
+#define AXP192_LDO2_ENABLE_REG_VAL     (0x04u)
+
+/* LDO3 parameters */
+#define AXP192_LDO3_VOLTAGE_REG_MASK   (0x0Fu)
+#define AXP192_LDO3_VOLTAGE_REG_BITPOS (0u)
+#define AXP192_LDO3_VOLTAGE_RANGE_MAX  (AXP192_LDO3_VOLTAGE_REG_MASK)
+#define AXP192_LDO3_VOLTAGE_RANGE_MIN  (0x00u)
+#define AXP192_LDO3_VOLTAGE_MIN_UV     (1800000u)
+#define AXP192_LDO3_VOLTAGE_MAX_UV     (3300000u)
+#define AXP192_LDO3_VOLTAGE_STEP_UV    (100000u)
+#define AXP192_LDO3_ENABLE_REG_MASK    (0x08u)
+#define AXP192_LDO3_ENABLE_REG_VAL     (0x08u)
+
+struct regulator_axp192_desc {
+	uint8_t enable_reg;
+	uint8_t enable_mask;
+	uint8_t enable_val;
+	uint8_t vsel_reg;
+	uint8_t vsel_mask;
+	uint8_t vsel_bitpos;
+	int32_t max_ua;
+	uint8_t num_ranges;
+	const struct linear_range *ranges;
+};
+
+struct regulator_axp192_common_config {
+	struct i2c_dt_spec i2c;
+};
+
+struct regulator_axp192_config {
+	struct regulator_common_config common;
+	const struct regulator_axp192_desc *desc;
+	const struct device *mfd;
+	LOG_INSTANCE_PTR_DECLARE(log);
```

Comment:
local compliance check always reports a weird error:
```
Running Checkpatch       tests in /Users/marteng/Projekte/zephyrproject/zephyr ...
1 checks failed
ERROR   : Test Checkpatch failed:
LINE_SPACING: Missing a blank line after declarations
File:drivers/regulator/regulator_axp192.c
Line:105
```
Any idea how to fix this?!?
GitHub task is still pending. Maybe just an issue local execution?

Reply:
Fixed